### PR TITLE
Blog Image captions and banner fix

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
1) Added styles for including captions for figures in `/blogs`.
Screenshot 2022-07-18 at 1 31 19 AM

2) Changed `poster` for the MeshMap video on banner.
3) Prevent flashing in between banners on refresh.

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 487f9557498fa71ce493e67ac78d0e45637ab628 at: https://62d47ac25eb1c41b1f2404cf--layer5.netlify.app

File: src/collections/blog/Blog.style.js [hunk 1]
```diff
@@ -207,6 +207,52 @@ export const BlogWrapper = styled.div`
...
+//   <img src="" />
+//   <figcaption>
+//     <p>This is text for the image</p>
+//   </figcaption>
+// </figure>
+=======================*/
+
+  figure.imgWithCaption {
+    margin-bottom: 1rem;
+    box-shadow: 0px 5px 10px 1px rgba(0, 179, 159, 0.5);
```
Comment (warunicorn19) [inline src/collections/blog/Blog.style.js:225]: Dont hard code the colors please

File: src/pages/index.js [hunk 2]
```diff
@@ -23,6 +23,14 @@ import CloudNativeLeaders from "../sections/Home/Cloud-Native";
 import SoSpecial from "../sections/Home/So-Special-Section";
 
 const IndexPage = () => {
+  const [mounted, setMounted] = React.useState(false);
+  React.useEffect(() => {
+    setMounted(true);
+  }, []);
+  if (!mounted) {
+    return <div style={{ visibility: "hidden" }}>Prevent Flash</div>;
```
Comment (warunicorn19) [inline src/pages/index.js:31]: Why is this required?

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 2 above)
Comment (debo19) [reply src/pages/index.js:31]: I was trying to prevent this flashing effect on refresh. Saw this approach on #2910 

https://user-images.githubusercontent.com/85789734/179425531-9645e9ba-19dc-43e0-a952-81538379a6ec.mov

Comment (warunicorn19) [reply src/pages/index.js:31]: Seems a little hacky 🤔

File: src/collections/blog/Blog.style.js (hunk 1 above)
Comment (debo19) [reply src/collections/blog/Blog.style.js:225]: This uses a reduced opacity so I cannot use the theme color just yet. We were discussing modifying the themes file to consider opacity as well but that's not implemented yet.

**Comment by leecalcote** [comment]
Screenshot from an iPhone. After a long wait, the video doesn’t start. 

088DA1A9-94AC-42C5-B66F-3C591585E882

File: src/pages/index.js (hunk 2 above)
Comment (leecalcote) [reply src/pages/index.js:31]: That blip/flash behavior is even worse, though.

**Review by leecalcote — APPROVED** [review]
